### PR TITLE
zh-trans zh/docs/tasks/configure-pod-container/static-pod.md

### PR DESCRIPTION
**Pull request by helight**
zh-trans zh/docs/tasks/configure-pod-container/static-pod.md
ref: https://github.com/k8smeetup/website-tasks/issues/3018

### CODE REVIEW

**Comment by helight** [comment]
@chenrui333 @tengqm  could you guys help to review this. thanks

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+{{< note >}}
+要确保 kubelet 在 API 服务上有创建镜像 Pod 的权限。如果没有，创建请求会被 API 服务拒绝。可以看[Pod安全策略](/docs/concepts/policy/pod-security-policy/)。
+{{< /note >}}
+
+<!--
+{{< glossary_tooltip term_id="label" text="Labels" >}} from the static Pod are
+propagated into the mirror Pod. You can use those labels as normal via
+{{< glossary_tooltip term_id="selector" text="selectors" >}}, etc.
+-->
+静态 Pod 上的{{< glossary_tooltip term_id="label" text="Labels" >}} 被传到镜像 Pod。 你可以通过 {{< glossary_tooltip term_id="selector" text="selectors" >}} 使用这些标签，比如。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/static-pod.md]: text="labels" -> text="标签"

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: text="selectors" -> text="选择算符"

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+*Static Pods* are managed directly by the kubelet daemon on a specific node,
+without the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+observing them.
+Unlike Pods that are managed by the control plane (for example, a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}});
+instead, the kubelet watches each static Pod (and restarts it if it crashes).
+-->
+
+*静态 Pod* 在指定的节点上由 kubelet 守护进程直接管理，不需要 {{< glossary_tooltip text="API 服务" term_id="kube-apiserver" >}} 监管。
+不像 Pod 是由控制面管理的（例如，{{< glossary_tooltip text="Deployment" term_id="deployment" >}}）；相反 kubelet 监视每个静态 Pod（在它奔溃之后重新启动）。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+If you are running clustered Kubernetes and are using static
+Pods to run a Pod on every node, you should probably be using a
+{{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}
+instead.
+{{< /note >}}
+-->
+
+静态 Pod 永远都会绑定到一个指定节点上的 {{< glossary_tooltip term_id="kubelet" >}}。
+
+kubelet 会通过 Kubernetes API 服务为每个静态 Pod 自动重试创建一个 {{< glossary_tooltip text="镜像 Pod" term_id="mirror-pod" >}}。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+这意味着节点上运行的静态 Pod 对 API 服务来说是不可见的。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+
+{{< note >}}
+如果你在运行一个 Kubernetes 集群，并且在每个节点上都运行一个静态 Pod，应该要用 {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}} 替代这种方式。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,
+and that your nodes are running the Fedora operating system.
+Instructions for other distributions or Kubernetes installations may vary.
+-->
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+这篇文章的前提是你在用 {{< glossary_tooltip term_id="docker" >}} 来运行 Pod，并且你的节点是运行在 Fedora 操作系统。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+    ssh my-node1
+    ```
+<!--
+2. Choose a directory, say `/etc/kubelet.d` and place a web server Pod definition there, e.g. `/etc/kubelet.d/static-web.yaml`:
+-->
+
+2. 选择一个目录，比如在 `/etc/kubelet.d` 目录来保存 web 服务 Pod 的定义文件， `/etc/kubelet.d/static-web.yaml`：
+
+    ```shell
+    # Run this command on the node where kubelet is running
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>"
+    ```
+<!--
+3. Restart the kubelet. On Fedora, you would run:
+-->
+
+1. 重启 kubelet。在 Fedora 上运行如下命令：
+
+    ```shell
+    # Run this command on the node where the kubelet is running
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+    或者在 [Kubelet配置文件](/docs/tasks/administer-cluster/kubelet-config-file)中添加 `staticPodPath: <目录>`字段。
+
+<!--
+1. Restart the kubelet. On Fedora, you would run:
+-->
+
+1. 重启 kubelet。Fedora 上使用下面的命令：
+
+    ```shell
+    # Run this command on the node where the kubelet is running
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: accept

**Comment by tengqm** [comment]
Thanks for the fixes.
/lgtm

**Pull request merged**